lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -26,115 +26,6 @@ module Rails
-            (send nil? {:assert_predicate :assert_not_predicate :refute_predicate} $_ ${sym} $_?)
-          PATTERN
-
-          # @!method minitest_match(node)
-          def_node_matcher :minitest_match, <<~PATTERN
-            (send nil? {:assert_match :refute_match} $_ $_ $_?)
-          PATTERN
-
-          # @!method minitest_nil(node)
-          def_node_matcher :minitest_nil, <<~PATTERN
-            (send nil? {:assert_nil :assert_not_nil :refute_nil} $_ $_?)
-          PATTERN
-
-          # @!method minitest_empty(node)
-          def_node_matcher :minitest_empty, <<~PATTERN
-            (send nil? {:assert_empty :assert_not_empty :refute_empty} $_ $_?)
-          PATTERN
-
-          def on_send(node) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
-            minitest_equal(node) do |expected, actual, failure_message|
-              on_assertion(node, EqualAssertion.new(expected, actual,
-                                                    failure_message.first))
-            end
-
-            minitest_instance_of(node) do |expected, actual, failure_message|
-              on_assertion(node, InstanceOfAssertion.new(expected, actual,
-                                                         failure_message.first))
-            end
-
-            minitest_predicate(node) do |subject, predicate, failure_message|
-              next unless predicate.value.end_with?('?')
-
-              on_assertion(node, PredicateAssertion.new(predicate, subject,
-                                                        failure_message.first))
-            end
-
-            minitest_includes(node) do |collection, expected, failure_message|
-              on_assertion(node, IncludesAssertion.new(expected, collection,
-                                                       failure_message.first))
-            end
-
-            minitest_match(node) do |matcher, actual, failure_message|
-              on_assertion(node, MatchAssertion.new(matcher, actual,
-                                                    failure_message.first))
-            end
-
-            minitest_nil(node) do |actual, failure_message|
-              on_assertion(node, NilAssertion.new(nil, actual,
-                                                  failure_message.first))
-            end
-
-            minitest_empty(node) do |actual, failure_message|
-              on_assertion(node, EmptyAssertion.new(nil, actual,
-                                                    failure_message.first))
-            end
-          end
-
-          def on_assertion(node, assertion)
-            preferred = assertion.replaced(node)
-            add_offense(node, message: message(preferred)) do |corrector|
-              corrector.replace(node, preferred)
-            end
-          end
-
-          def message(preferred)
-            format(MSG, prefer: preferred)
-          end
-
           # :nodoc:
           class BasicAssertion
```

Comment:
Should we consider adding empty methods to `BasicAssertion` to indicate the interface?
```rb
def negated?
  raise NotImplementedError
end

def assertion
  raise NotImplementedError
end
```